tests/e2e/k8s/util/fabric.go

```diff
@@ -38,6 +39,8 @@ type PeerConfig struct {
 	DataplaneType string
 	// Dataplanes is the number of dataplane instances.
 	Dataplanes uint16
+	// LogLevel is the log level.
+	LogLevel string
```

Comment:
I don't think we need this.
You can read the "DEBUG" "CICD" env variables at `Fabric::generateK8SYAML`, and pass it on directly to `platform.K8SConfig`.